### PR TITLE
Update e2ee guide

### PR DESCRIPTION
**Pull request by Sorunome**
Update the e2ee guide to contain bootstrapping (and hopefully soon online key backup)

### CODE REVIEW

File: gatsby/content/docs/e2ee_cross_signing.mdx [hunk 1]
```diff
@@ -1196,3 +1196,223 @@ and use that to verify and sign yourself:
...
+To set the newly generated key as default key, simply set the `m.secret_storage.default_key` account
+data to the key ID (*NOT* the full account data type):
+
+```json
+{
+  "key": "<keyId>"
+}
+```
+
+### Bootstrappning cross-signing
```
Comment (mirukana) [inline gatsby/content/docs/e2ee_cross_signing.mdx:1286]:
```suggestion
### Bootstrapping cross-signing
```

File: gatsby/content/docs/e2ee_cross_signing.mdx [hunk 2]
```diff
@@ -1196,3 +1196,223 @@ and use that to verify and sign yourself:
 4. Sign your own master key with your own device key.
 5. Upload the signatures.
 6. Fetch all entries from SSSS that you want to cache and cache them.
+
```
Comment (MadLittleMods) [inline gatsby/content/docs/e2ee_cross_signing.mdx:1199]: @Sorunome The current state of the content looks good as an iterative update. Is the PR in a draft state for a particular blocker?

**Comment by Sorunome** [comment]
pretty sure the only reason was that soru wanted to add more, but guess that could be an iterative next PR, too

26. 5. 2021 3:31:46 Eric Eastwood ***@***.***>:

File: gatsby/content/docs/e2ee_cross_signing.mdx (hunk 1 above)
Comment (MadLittleMods) [reply gatsby/content/docs/e2ee_cross_signing.mdx:1286]: @Sorunome Mind accepting this change? (I don't have access for your fork)

Comment (MadLittleMods) [reply gatsby/content/docs/e2ee_cross_signing.mdx:1286]: Fixed typo in https://github.com/matrix-org/matrix.org/commit/cd572b2eaa9d4b2d19fa78aa99c37e5c5c0a848c

**Comment by MadLittleMods** [comment]
Merged 🚀 Thanks for the updates @Sorunome 🦦

**Pull request merged**